### PR TITLE
6pm c office hours tutor assignment

### PR DESCRIPTION
**Pull request by kourosh-safari**
Implements office hours view, create, and delete (for various user roles).

### CODE REVIEW

File: javascript/src/main/App.js [hunk 1]
```diff
@@ -43,8 +47,11 @@ function App() {
           <AuthorizedRoute path="/courses/new" exact component={NewCourse} authorizedRoles={["admin"]} />
           <AuthorizedRoute path="/courses/edit/:courseId" exact component={EditCourse} authorizedRoles={["admin"]} />
           <AuthorizedRoute path="/tutorAssignment" exact component={TutorAssignment} authorizedRoles={["admin", "member"]} />
-          <AuthorizedRoute path="/tutorAssignment/new" exact component={NewTutorAssignment} authorizedRoles={["admin", "member"]} />
+          <AuthorizedRoute path="/tutorAssignment/new" exact component={NewTutorAssignment} authorizedRoles={["admin", "instructor"]} />
```
Comment (sethvanb) [inline javascript/src/main/App.js:50]: want member not instructor

File: javascript/src/main/components/Courses/CourseTable.js [hunk 2]
```diff
@@ -3,8 +3,6 @@ import BootstrapTable from 'react-bootstrap-table-next';
 import { Button } from "react-bootstrap";
 import { useHistory } from "react-router-dom";
 
-import { buildDeleteCourse } from "main/services/Courses/CourseService";
```
Comment (sethvanb) [inline javascript/src/main/components/Courses/CourseTable.js:6]: Why did we delete this

**Review by sethvanb — CHANGES_REQUESTED** [review]
You have comments in your code so take them out.

**Review by sethvanb — DISMISSED** [review]

**Comment by sethvanb** [comment]
LGTM

**Review by pconrad — CHANGES_REQUESTED** [review]
The .DS_Store files need to be removed.

I suggest also adding `.DS_Store` to the top level `.gitignore` after you delete these.

They are inflating the number of files changed, and cluttering up the repo.

Be sure to *always* do a `git status` after doing a `git add .` to make sure you haven't picked up stray files that don't belong in the repo.

File: javascript/src/test/components/OfficeHours/OfficeHourTable.test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  }]
+
+
+
+  test("renders without crashing", () => {
+    render(<OfficeHourTable officeHours = {sampleOfficeHour}/>);
+  });
+
+/*  test("Test delete button calls func", () => {
```
Comment (pconrad) [inline javascript/src/test/components/OfficeHours/OfficeHourTable.test.js:25]: Why is this test commented out?   If the delete function hasn't been written yet, then we can delete this test.  If it has been written and isn't being tested, we should test it (But then you'd probably get coverage problems, right?)

**Review by pconrad — CHANGES_REQUESTED** [review]
One small place where there's some commented out code, and one .DS_Store file that need to be removed.  Other than that, nice work.

File: javascript/src/test/components/OfficeHours/OfficeHourTable.test.js (hunk 3 above)
Comment (kourosh-safari) [reply javascript/src/test/components/OfficeHours/OfficeHourTable.test.js]: I believe test got moved to a more applicable location for testing.

**Review by sethvanb — DISMISSED** [review]
LGTM

**Review by sethvanb — DISMISSED** [review]
LGTM

**Review by sethvanb — DISMISSED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

There is that one comment about AUTO, but hey, we'll fix it later :-)

**Comment by pconrad** [comment]
Team already exceeded max point total, so no additional points awarded.  But this would probably 20 to 40 points depending.

**Pull request merged**